ci/do_ci.sh

```diff
@@ -214,7 +214,7 @@ case $CI_TARGET in
         if [[ -n "$ENVOY_API_ONLY" ]]; then
             exit 0
         fi
-        ;&
+        ;;
```

Comment:
this isnt a typo its intentional - it makes running `do_ci.sh api` fallthrough to also build the `api.go` target

Reply:
Oh wow 🤯